DOCS: Re-organize controls section to conform to content model, DOCF-6394 (#2128)
Unity-Technologies/InputSystem: Packages/com.unity.inputsystem/Documentation~/noisy-controls.md

> # Noisy controls

Reviewer: Would something like "Handling noisy data from controls" be a better way of describing this?

Author: It's more a description of how the input system automatically handles noisy controls, so it doesn't really tell you how to handle them. It's just info to be aware of rather than something you have to do yourself.

> Noisy controls are those that can change value without any actual or intentional user interaction required. A good example of this is an accelerometer or gyroscope sensor in a cellphone. Even if the cellphone is at rest, there are usually fluctuations in the control's value readings. Another example are orientation readings from a head-mounted display.

Reviewer: ```suggestion
Noisy controls are those that can change value without any actual or intentional user interaction required. For example, they gyroscope sensor in a cellphone provides noisy input data because even if the cellphone is at rest, there are usually fluctuations in the control's value readings. Another example are orientation readings from a head-mounted display.
```

Some built-in control types are marked as **noisy**. You can query this using the  [`InputControl.noisy`](../api/UnityEngine.InputSystem.InputControl.html#UnityEngine_InputSystem_InputControl_noisy) property, or by inspecting the control types in the [Input Debugger window](the-input-debugger-window.md).

If a control is marked as noisy:

- The control is not considered for [interactive rebinding](interactive-rebinding.md). [`InputActionRebindingExceptions.RebindingOperation`](../api/UnityEngine.InputSystem.InputActionRebindingExtensions.RebindingOperation.html) ignores the control by default (you can bypass this using [`WithoutIgnoringNoisyControls`](../api/UnityEngine.InputSystem.InputActionRebindingExtensions.RebindingOperation.html#UnityEngine_InputSystem_InputActionRebindingExtensions_RebindingOperation_WithoutIgnoringNoisyControls)).

> - The Input System performs additional event filtering to filter out noise, then calls [`InputDevice.MakeCurrent`](../api/UnityEngine.InputSystem.InputDevice.html#UnityEngine_InputSystem_InputDevice_MakeCurrent) if any non-noise values cause the control to change state. If an input event for a Device contains no state change on a Control that is not marked noisy, then the Device will not be made current based on the event. This avoids, for example, a plugged in PS4 controller constantly making itself the current gamepad ([`Gamepad.current`](../api/UnityEngine.InputSystem.Gamepad.html#UnityEngine_InputSystem_Gamepad_current)) due to its sensors constantly feeding data into the system.

Reviewer: ```suggestion
- The Input System performs additional event filtering to filter out noise, then calls [`InputDevice.MakeCurrent`](../api/UnityEngine.InputSystem.InputDevice.html#UnityEngine_InputSystem_InputDevice_MakeCurrent) if any non-noise values cause the control to change state. If an input event for a device doesn't contain a state change on a non-noisy control, then the Input System doesn't mark the device as current based on the event. This avoids, for example, a plugged in PS4 controller constantly making itself the current gamepad ([`Gamepad.current`](../api/UnityEngine.InputSystem.Gamepad.html#UnityEngine_InputSystem_Gamepad_current)) due to its sensors constantly feeding data into the system.
```

There were a lot of negatives here that were making it hard to parse

> - When the application loses focus and Devices are [reset](Devices.md#device-resets) as a result, the state of noisy Controls will be preserved as is. This ensures that sensor readinds will remain at their last value rather than being reset to default values.

Reviewer: ```suggestion
- When the application loses focus and devices are [reset](Devices.md#device-resets) as a result, the state of noisy controls are preserved. This ensures that sensor readings remain at their last value rather than being reset to default values.
```
